pennylane/gradients/__init__.py

```diff
@@ -131,7 +122,10 @@ def circuit(weights):
 Transforming QNodes
 -------------------
 
-Alternatively, quantum gradient transforms can be applied manually to QNodes.
+Alternatively, quantum gradient transforms can be applied manually to QNodes. This is not
+recommended because PennyLane must compute the classical Jacobian of the parameters and multiply it with
+the quantum Jacobian, we recommend using the `diff_method` kwargs with your favorite machine learning

```

Comment:
```suggestion
the quantum Jacobian, we recommend using the ``diff_method`` kwargs with your favorite machine learning
```
don't forget that ReST needs double backticks!